xbmc/games/addons/input/GameClientInput.cpp

```diff
@@ -471,6 +471,7 @@ bool CGameClientInput::OpenKeyboard(const ControllerPtr& controller,
   {
     m_keyboard =
         std::make_unique<CGameClientKeyboard>(m_gameClient, controller->ID(), keyboard.get());
+    m_keyboard->SetSource(std::move(keyboard));
```

Comment:
GCC: `xbmc/games/addons/input/GameClientInput.cpp:474:36: warning: redundant move in initialization [-Wredundant-move]`
clang-tidy: `xbmc/games/addons/input/GameClientInput.cpp:474:27: warning: std::move of the const variable 'keyboard' has no effect; remove std::move() or make the variable non-const [performance-move-const-arg]`